css/shared/colors.css

```diff
@@ -0,0 +1,2342 @@
+/* text colors */
+
+/* pink text colors */
+.text-pink-100 {
+  color: var(--pink-100);
+}
+.text-pink-200 {
+  color: var(--pink-200);
+}
+.text-pink-300 {
+  color: var(--pink-300);
+}
+.text-pink-400 {
+  color: var(--pink-400);
+}
+.text-pink-500 {
+  color: var(--pink-500);
+}
+.text-pink-600 {
+  color: var(--pink-600);
+}
+.text-pink-700 {
+  color: var(--pink-700);
+}
+.text-pink-800 {
+  color: var(--pink-800);
+}
+.text-pink-900 {
+  color: var(--pink-900);
+}
+
+/* rose text colors */
+.text-rose-100 {
+  color: var(--rose-100);
+}
+.text-rose-200 {
+  color: var(--rose-200);
+}
+.text-rose-300 {
+  color: var(--rose-300);
+}
+.text-rose-400 {
+  color: var(--rose-400);
+}
+.text-rose-500 {
+  color: var(--rose-500);
+}
+.text-rose-600 {
+  color: var(--rose-600);
+}
+.text-rose-700 {
+  color: var(--rose-700);
+}
```

Comment:
Easier approach is to create all variable in root 
:Root{
Var_name : VAR(#ffff)}